.pylintrc

```diff
@@ -1,2 +1,5 @@
 [TYPECHECK]
 generated-members=logging.verbose
+
+[main]
+max-module-lines=1100
```

Comment:
The `zap` module needs some refactoring to reduce its code complexity. For now, I've temporarily increased the threshold as a quick fix

Reply:
Disabled the check directly within the module, rather than globally